CHANGELOG-1.2.md

```diff
@@ -0,0 +1,33 @@
+# CHANGELOG of v1.2
+
+Previous change logs can be found at [CHANGELOG-1.1](https://github.com/opencurve/curve/blob/master/CHANGELOG-1.1.md)
+
+## new features
+
+- When add a new pool, default new logic pool is DENY, need tool to enable it.
+- When io error occurs, whether nebd-server discards the corresponding rpc request depends on configuration setting(discard default).
+- When client read a unallocated space, it does not allocate the segment to improve the space utilization.
+- Add data stripe feature.
+
+
+## optimization
+
+- Optimize ansible script, optimize build script, optimize log printing.
+- Translate some document and code comment from chinese to english.
+- Curve_ops_tool statistics chunkserver capacity by pool.
+- Add a script for k8s to attach curve volume.
+- curve_ops_tool improve:
+    - Check copyset consistency between chunkserver and copyset.
+    - Support to take a snapshot of copyset.
+    - Support to list the volume on copyset where majority copy is failed.
+- Clean up the unit test temporary folder.
+
+## bug fix
+
+- Fix clone delete bug.
+- Nbd unmap need wait thread exit.
+- Mds need check file attach status.
+- Fixed when disk fails, copyset reports an error, but sometimes the chunkserver does not exit.
+- Fixed the direct_fd leak problem when wal write disk.
+- Fixed the atomicity problem of GetFile when not using the file pool.
+- Fixed in cluster_basic_test mds start error on InitEtcdClient(), and produced coredump
```

Comment:
Fixed in cluster_basic_test mds start error on InitEtcdClient() which  produced coredump?